pumps/prometheus.go

```diff
@@ -178,12 +174,39 @@ func (p *PrometheusPump) Init(conf interface{}) error {
 	go func() {
 		log.Fatal(http.ListenAndServe(p.conf.Addr, nil))
 	}()
-	p.log.Info(p.GetName() + " Initialized")
+	p.log.WithField("disabled_metrics", disabledMetrics).Info(p.GetName() + " Initialized")
 
 	return nil
 }
 
-//InitCustomMetrics initialise custom prometheus metrics based on p.conf.CustomMetrics and add them into p.allMetrics
+func (p *PrometheusPump) initBaseMetrics() []string {
```

Comment:
@hsyed-dojo I don't think we should return the disabled metrics here, it doesn't make sense that the initBaseMetrics func return that IMO.

Reply:
If one ops engineer goes and disables a metric and another ops engineer expects that metric to still be active in an environment I suspect it would help them to see a log message indicating that the metric was disabled. I guess they can inspect the config 🤷‍♂️ .

Will remove it.